frontend/src/templates/Field/Table/ColumnCell.tsx

```diff
@@ -67,11 +73,15 @@ const ShortTextColumnCell = ({
 
 const DropdownColumnCell = ({
   schema,
+  responseMode,
   inputName,
   colorTheme,
 }: FieldColumnCellProps<DropdownColumnBase>) => {
   const { control } = useFormContext<TableFieldInputs>()
-  const rules = useMemo(() => createDropdownValidationRules(schema), [schema])
+  const rules = useMemo(
+    () => createDropdownValidationRules(schema, responseMode),
```

Comment:
same as above
We can't pass in `responseMode` here, `createDropdownValidationRules` accepts a schema and a `disableRequiredValidation` boolean val

Reply:
https://github.com/opengovsg/FormSG/commit/cafd0762a90aa091ba634f1d705b70f0cafbf481